src/vaadin-grid.html

```diff
@@ -714,6 +715,10 @@
       notifyResize() {
         super.notifyResize();
       }
+
+      _heightByRowsChanged() {
+        this.notifyResize();
```

Comment:
Should we guard against switching from `undefined` to `false` (default value)?

Reply:
Good point, we can avoide one unnecessary notifyResize. Fixed.